src/stores/RoomStore.ts

```diff
@@ -82,8 +84,13 @@ export class RoomStore implements RoomViewModel {
   private _pomodoroTimerEventDate?: Date = undefined;
   private _pomodoroProperty?: PomodoroTimerProperty = undefined;
   private _blacklist: BlockedUser[] = [];
-
   private _kicked: boolean = false;
+  private _videoDeviceList: MediaDeviceInfo[] = [];
+  private _audioDeviceList: MediaDeviceInfo[] = [];
+  private _speakerDeviceList: MediaDeviceInfo[] = [];
+  private _currentVideoDeviceId: string | undefined = "";
+  private _currentAudioDeviceId: string | undefined = "";
+  private _currentSpeakerDeviceId: string = "";
```

Comment:
초기값은 undefined로 해야할것같네요.
```suggestion
  private _currentVideoDeviceId: string | undefined = undefined;
  private _currentAudioDeviceId: string | undefined = undefined;
  private _currentSpeakerDeviceId: string | undefined = undefined;
```